source/documentation/troubleshooting/index.md

```diff
@@ -29,12 +29,131 @@ cf events APPNAME
 
 Running ``cf events`` shows you when an app starts, stops, restarts, or crashes (including error codes). The output is often easier to interpret than the output of ``cf logs``.
 
+### Third-Party Log Management Services
+
+The default Cloud Foundry logging system stores a limited amount of logging information for a defined time. You can retain more logging information for a longer period of time by setting up a third-party log management service. Instructions on how to set up a log management service can be found [here](https://docs.cloudfoundry.org/devguide/services/log-management.html) [external link]. The next section details how to set up an example log management service, [Logit.io](https://logit.io/) [external link].
```

Comment:
@jonathanglassman I'm wondering if this could be misleading:

> You can retain more logging information for a longer period of time by setting up a third-party log management service.

The 'more' could be confusing: it implies that a third-party system gets more logs than `cf logs --recent` option. But the key benefit is that logs are retained for longer. I wonder if removing the 'more' would work:

> You can retain logging information for a longer period of time by setting up a third-party log management service.

What do you think?

Reply:
I think you're right - I'll remove. Thanks for the review.